importer/PeEs.py

```diff
@@ -0,0 +1,90 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class PeEs(Monument):
+
+    def set_location(self):
+        """Set the Location, using linked article."""
+        if self.has_non_empty_attribute("localidad"):
+            if utils.count_wikilinks(self.localidad) == 1:
+                loc_q = utils.q_from_first_wikilink("es", self.localidad)
+
+            if loc_q is not None:
+                self.add_statement("location", loc_q)
+
+    def update_labels(self):
+        spanish = utils.remove_markup(self.monumento)
+        self.add_label("es", spanish)
+
+    def update_descriptions(self):
+        eng = "cultural heritage site in Peru"
+        self.add_description("en", eng)
+
+    def set_adm_location(self):
+        """
+        Set the Admin Location.
+
+        Use the linked Municipality first, checking
+        against external list.
+        If failed, use the Region iso code, which is a
+        bigger unit.
+        """
+        adm_q = None
+        municip_dic = self.data_files["municipalities"]
+        reg_dic = self.data_files["regions"]
+
+        municip_q = utils.q_from_first_wikilink("es", self.municipio)
+        if municip_q in [x["item"] for x in municip_dic]:
+            adm_q = municip_q
```

Comment:
Add to report already if this fails (since iso is less specific)

Reply:
Done.